styles/main.css

```diff
@@ -1,96 +1,103 @@
 body {
-    background-color: #62D8BB; 
-    font-family: 'Open Sans Condensed', sans-serif;  
-    }
-   
-   h1 {color: transparent; 
-       font-family:'Alegreya', sans-serif;
-       text-shadow: 1px 1px 1px rgba(0,0,0,0.2);
-      -webkit-background-clip: text;
-      background-clip: text; 
-       opacity: 1.0;    
-       font-family: Alegreya, sans-serif;
-       background-color: rgb(0,0,0);
-       background-image: linear-gradient(0deg, rgba(0,0,0,1) 18%, rgba(2,121,113,1) 65%, rgba(0,144,135,1) 100%);   }
-   
-   h2 {color: transparent;  
-       font-family: 'Alegreya', sans-serif;
-       text-shadow: 1px 1px 1px rgba(0,0,0,0.2);
-       opacity: 1.0;
-       -webkit-background-clip: text;
-        background-clip: text;
-       background-color: rgb(0,0,0);
-       background-image: linear-gradient(0deg, rgba(0,0,0,1) 18%, rgba(2,121,113,1) 65%, rgba(0,144,135,1) 100%);
-
-   }
-   
-   a {
-       color: #EE3333;
-       text-decoration: none;
-   }
-   
-   nav a  {
-       color: #007700;
-   }
-   
-   .byline {
-       color: rgba(100,10,10,0.5);
-       font-size: 70%
-   } 
-   
-   #page-copyright {
-       color: #B14600;
-       letter-spacing: 3px;
-       text-transform: uppercase;
-   }
-   
-   .bio p {
-       text-indent: 3em;
-   }
-   
-   footer {
-       text-align: center;
-       font-size: 12px;
-       margin-bottom: 2em;
-   }
-   header ul{
-       list-style-type: none;
-       margin: 0;
-       padding: 0;
-   }
-   
-   ul {
-       list-style-type: lower-roman;
-   }
-   
-   .width-wrap {
-       margin-right: auto;
-       margin-left: auto;
-       max-width: 1024px;
-       padding-right: 2em;
-       padding-left: 2em;
-       margin-bottom: 3em;
-       border-left: 1px solid rgba(0,0,0,0.3);
-       border-right: 1px solid rgba(0,0,0,0.3);
-       border-bottom: 1px solid rgba(0,0,0,0.3);
-       background-color: #53A8AA;
-       box-shadow: 0 0 4px  rgba(0,0,0,0.2);
-         
-   }
-   
-   figure img{
-       box-shadow: 2px 2px 3px rgba(0,0,0,0.5);
-       
-       
-   }
-   
-   hr {

```

Comment:
Incorrect selector: you need to specify `footer hr` to get ONLY `hr` elements in the footer